browser/src/Services/Learning/Tutorial/Tutorials/InlineFindingTutorial.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * TutorialManager
+ */
+
+import * as React from "react"
+
+import { ITutorial, ITutorialMetadata, ITutorialStage } from "./../ITutorial"
+// import { InitializeBufferStage, MoveToGoalStage } from "./../Stages"
+
+import * as Notes from "./../Notes"
+import * as Stages from "./../Stages"
+
+const Line1 = "Use 'f' to move to a character to the RIGHT of the cursor."
+const Line2 = "And use 'F' to move to a character to the LEFT."
+const Line3 = "'t' is like 'f' except it moves to one spot before the character."
+const Line4 = "And 'T' is like 'F' except it moves one spot after."
+const Line5 = "Awesome! You can also use ';' to repeat the last f, t, F, or T."
+const Line6 = "And use ',' to repeat the last f, t, F, or T in the opposite direction."
+
+export class InlineFindingTutorial implements ITutorial {
+    private _stages: ITutorialStage[]
+
+    constructor() {
+        this._stages = [
+            new Stages.SetBufferStage([Line1]),
+            new Stages.MoveToGoalStage("Move to the 'R' in 'RIGHT' using 'fR'", 0, 38),
+            new Stages.SetBufferStage([Line1, Line2]),
+            new Stages.MoveToGoalStage("Use 'j' to move down a line", 1, 38),
+            new Stages.MoveToGoalStage("Use 'F' to move LEFT to the goal", 1, 9),
```

Comment:
When running the tutorial, it's kind of awkward hitting `F` to move to the character `F`.  Can we move to a different character in that sentence?